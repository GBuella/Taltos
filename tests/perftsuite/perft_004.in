setboard 4k2r/8/8/8/8/8/8/4K3 w k - 0 1
echo 4k2r/8/8/8/8/8/8/4K3 w k - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
