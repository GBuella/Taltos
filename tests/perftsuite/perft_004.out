4k2r/8/8/8/8/8/8/4K3 w k - 0 1
5
75
459
8290
47635
899442
